src/components/Tasks/TaskEdit.tsx

```diff
@@ -11,28 +13,20 @@ import MenuItem from '@material-ui/core/MenuItem';
 import FormControl from '@material-ui/core/FormControl';
 import FormHelperText from '@material-ui/core/FormHelperText';
 import InputLabel from '@material-ui/core/InputLabel';
-import * as colors from '@material-ui/core/colors';
+import { red } from '@material-ui/core/colors';
+import FormLabel from '@material-ui/core/FormLabel';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import Autocomplete from '@material-ui/lab/Autocomplete';
 
 import IconDelete from '@material-ui/icons/Delete';
 import IconCancel from '@material-ui/icons/Clear';
 import IconSave from '@material-ui/icons/Save';
 
-import DateTimePicker from '../widgets/DateTimePicker';
-
-import ConfirmationDialog from '../widgets/ConfirmationDialog';
-import TimezonePicker from '../widgets/TimezonePicker';
-
-import { Location } from 'history';
-import { withRouter } from 'react-router';
```

Comment:
All of this unnecessary reordering makes it impossible to review. Please revert this and only change what you are actually changing.